CMakeLists.txt

```
@@ -1,14 +1,15 @@
 cmake_minimum_required(VERSION 3.16)
-project(AudioPlugins VERSION 1.0.0)
+project(AudioPlugin VERSION 1.0.0)
```

```suggestion
project(AudioPlugins VERSION 1.0.0)
```

Minor nitpick: The plural was actually here on purpose as a placeholder project name. The CMake project can contain multiple `juce_add_plugin` with independent product names (I'm imagining building multiple small FX 🔜) that share build config, and potentially source files too.

---

I was getting linker errors when the project name was `AudioPlugins`, which is why I changed it to `AudioPlugins`